apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/DateQuestionForm.tsx

```diff
@@ -0,0 +1,93 @@
+import { TSurvey, TSurveyDateQuestion } from "@formbricks/types/surveys";
+import { Button } from "@formbricks/ui/Button";
+import { Label } from "@formbricks/ui/Label";
+import { Input } from "@formbricks/ui/Input";
+import { PlusIcon, TrashIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import QuestionFormInput from "./QuestionFormInput";
+import { OptionsSwitcher } from "@formbricks/ui/QuestionTypeSelector";
+
+interface IDateQuestionFormProps {
+  localSurvey: TSurvey;
+  question: TSurveyDateQuestion;
+  questionIdx: number;
+  updateQuestion: (questionIdx: number, updatedAttributes: any) => void;
+  lastQuestion: boolean;
+  isInValid: boolean;
+}
+
+const dateOptions = [
```

Comment:
<img width="2064" alt="Screenshot 2023-11-21 at 18 36 50" src="https://github.com/formbricks/formbricks/assets/675065/82323a9a-b227-4ee2-ae4c-62999183e5fe">

When I already have a Date selected in the Preview, changing the formatting options on the left isn't changing the UI preview on the right. Is this possible to change?